changelog

```diff
@@ -37,6 +37,7 @@ Version 1.13.10+dev:
    * New wesnoth.unit_types[].advances_from getter.
  * Multiplayer:
    * Dark Forecast: Fixed broken faction and leader selection.
+   * Rename the Khalifate to the Dunefolk.
```

Comment:
I'd remove the second "the"; it sounds a little awkward to me.

Reply:
Fair enough.